async-opcua-client/src/session/connection.rs

```diff
@@ -1,42 +1,72 @@
 use std::{str::FromStr, sync::Arc};
 
+use crate::{
+    transport::{tcp::TransportConfiguration, Connector, ConnectorBuilder},
+    AsyncSecureChannel, ClientConfig, IdentityToken,
+};
```

Comment:
Why did you move the `use crate` block in between the `use std` and `use xxx (3rd party crates)` blocks? I think most Rust projects place the crates own packages last as do we in most (if not all) other files.

Reply:
I'll be honest. I spend zero brain-cycles on import ordering, just relying on rustfmt to deal with it. Interestingly I think rustfmt preserves import blocks if they have blank lines between them. What happened here is I removed a blank line, and it formatted `crate::` above `opcua*::`. Not sure why, really. I can revert it.